src/Java.Interop.Tools.JavaTypeSystem/Java.Interop.Tools.JavaTypeSystem.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(DotNetTargetFramework)</TargetFramework>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <LangVersion>12</LangVersion>
```

Comment:
Isn't this change moot with c1cd06dde871b468f6c729fdce496d2971b95003?

Reply:
Yes, this `.csproj` change should have been reverted, fixed.